api/typescript/src/bindings/env.d.ts

```diff
@@ -95,6 +103,8 @@ export class Env {
   cacheSet(key: string, value: Uint8Array, ttl: number, ident: number): number;
   cacheGet(key: string, ident: number): number;
   requestGetField(fieldType: FieldType, key: string, ident: number): number;
+  requestSetField(fieldType: FieldType, key: string, value: Uint8Array, ident: number): number;
+  respSetHeader(key: string, value: Uint8Array, ident: number): void;
```

Comment:
Hmm any reason `value` does not simply take a string here? I would think that's more appropriate for header values :thinking: 

Rust seems to take also `set_header(key: &str, val: &str)`.